Week-2/Exercise 2.1/base.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bikester</title>
+    <link rel="icon" type="image/png" sizes="64x64" href="favicon.ico" />
+    <link rel="stylesheet" href="styl.css" />
+
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
what is the use of this if https://fonts.googleapis.com is already there

Reply:
Yes Jesus , Those are Extra Line ,  We can Comment them.